changelog/61696.fixed

```diff
@@ -0,0 +1 @@
+Deprecated old style Windows Group Policy names
```

Comment:
This is a removal not a deprecation. Can you update the file extension to be `removed` and update the description from Deprecated to Removed.

Reply:
Fixed